src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import classnames from 'classnames'
+
+// Import individual SVGs
+import AccessibilityNewSvg from 'uswds/dist/img/usa-icons/accessibility_new.svg'
+
+interface USWDSIconProps {
+  focusable?: boolean
+  role?: string
+  size?: 3 | 4 | 5 | 6 | 7 | 8 | 9
+  className?: string
+}
+
+type IconProps = USWDSIconProps & JSX.IntrinsicElements['svg']
+
+function makeUSWDSIcon(Component: React.ComponentType<IconProps>) {
+  return (props: IconProps) => {
+    const {
+      size,
+      className,
+      focusable = false,
+      role = 'img',
+      ...iconProps
+    } = props
+
+    const classes = classnames(
+      'usa-icon',
+      {
+        [`usa-icon--size-${size}`]: size !== undefined,
+      },
+      className
+    )
+
+    const finalProps = {
+      className: classes,
+      focusable,
+      role,
+      ...iconProps,
+    }
+
+    return <Component {...finalProps} />
+  }
+}
+
+export const AccessibilityNew: React.ComponentType<IconProps> =
+  makeUSWDSIcon(AccessibilityNewSvg)
```

Comment:
And so the plan would be to create one of these for each one of the icons from uswds, we at one point we had in the `icons.ts` file? Or did you want to try to automate the generation of this somehow?

Reply:
as much as I would like to automate this, it's taking a whole new effort to figure out and I'm inclined to just do the cheap & easy thing for now (knowing new icons won't be added unless the USWDS version changes). I will create a followup issue for automating this, possibly using a webpack plugin or script??

Reply:
I'm totally on board with a manual process!